src/main/kotlin/com/exactpro/example/flow/RegisterTradeFlow.kt

```diff
@@ -21,6 +23,8 @@ class RegisterTradeFlow(val fixMessage: FixMessage, val ccp: Party)
             : ProgressTracker.Step("Generating transaction.")
         object VERIFYING_TRANSACTION
             : ProgressTracker.Step("Verifying contract constraints.")
+        object EXCHANGING_IDENTITIES

```

Comment:
It should override `childProgressTracker()` and return `IdentitySyncFlow.Send`'s progress tracker from it